Puppetfile

```diff
@@ -51,7 +51,7 @@ mod 'maestrodev-wget',
   :git => 'https://github.com/maestrodev/puppet-wget',
   :ref => 'master'
 
-mod 'stackstorm-st2',                '0.12.2'
```

Comment:
That's correct, but they really should be since we don't have any other way to roll out those changes (we always pull down latest st2workroom master).